packages/react/kibana_context/common/kibana.jsonc

```diff
@@ -1,5 +1,5 @@
 {
-  "type": "shared-browser",
+  "type": "shared-common",
```

Comment:
have to change due to importing into common (shared-src) https://buildkite.com/elastic/kibana-pull-request/builds/263341#019417b6-599b-421e-92d3-0e19c1873654 .